runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessConfigureView/ProcessConfigureView.tsx

```diff
@@ -56,6 +79,12 @@ const ProcessConfigureView: VFC = () => {
         if (process?.isTriggerable) setTriggerable(process.isTriggerable);
     }, [process]);
 
+    useEffect(() => {
+        const subscription = processSubscriptions.find(sub => sub.user.id === user.id);
+        setCurrentProcessSubscription(subscription);
+        setSubscribed(Boolean(subscription));
+    }, [processSubscriptions]);
```

Comment:
imo backend should return to you ready to display a list of subscribers and not carry other subscribers that you should not know about. Unfortunately, I want you to move this logic to the orchestrator.